namespaces/live.cloud-platform.service.justice.gov.uk/prisoner-content-hub-development/resources/elasticsearch.tf

```diff
@@ -0,0 +1,22 @@
+module "content_hub_elasticsearch" {
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-elasticsearch?ref=3.9.1"
+  cluster_name           = var.cluster_name
+  application            = var.application
+  business-unit          = var.business-unit
+  environment-name       = var.environment-name
+  infrastructure-support = var.infrastructure-support
+  is-production          = var.is-production
+  team_name              = var.team_name
+  elasticsearch-domain   = "hub-search"
+  namespace              = var.namespace
+  elasticsearch_version  = "7.1"
+  instance_type          = "t2.small.elasticsearch"
+  irsa_enabled           = true
+  assume_enabled         = false
```

Comment:
noticed these two fields `irsa_enabled` `assume_enabled` and  were added as part of the migration that differ from dev

Reply:
Thats correct @andrewrlee , Those variables are needed as part of migration so please keep them